### PR TITLE
Submission assignment

### PR DESCRIPTION
**Pull request by yanistazi**

### CODE REVIEW

File: assignment/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import pandas as pd
+import natsort
```
Comment (axiezai) [inline assignment/tests/test_utils.py:3]: remove this line, not needed in this file, and not part of the pytest github action setup

**Comment by yanistazi** [comment]
updated without natsort

**Comment by yanistazi** [comment]
Cool ! Works now

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install pytest numpy pandas
+        python -m pip install pytest numpy pandas natsort
```
Comment (axiezai) [inline .github/workflows/main.yml:26]: TY for adding this. Was my mistake.

File: environment.yml [hunk 3]
```diff
@@ -15,3 +15,4 @@ dependencies:
     - natsort
     - sklearn
     - pytest
+prefix: /Users/taziy/anaconda3/envs/pytest3
```
Comment (axiezai) [inline environment.yml:18]: these prefixes are usually not useful for others =/ I tend to keep them away from my shared env files, so other people don't accidentally make a directory in their `/Users` with my username on it.

**Comment by axiezai** [comment]
All 4 tests passing,

Only real bothersome thing is the `pycache` and `ipynb_checkpoints` files, should be in your global gitignore for convenience TBH. 

100%. Good job!

**Pull request merged**